src/lib/media-query/breakpoints/break-point-registry.ts

```diff
@@ -30,6 +30,20 @@ export class BreakPointRegistry {
   }
 
   /**
+   * Accessor to sorted list used for
```

Comment:
Tried to give focus to both equally :) My understanding is that we'll want to review this library with the same rigor we give to Material, and we're often strict about even the small things just to maintain quality (even in comments). Apologies if it seems pedantic, just wanting to make sure I'm fair to all PRs